mantidproject/mantid
# Add plotting for sample logs

Add possibility to plot sample logs.

**To test:**
Check documentation, see if you can run according to the instructions
Using files from DocTests and SystemTests
```
import matplotlib.pyplot as plt
from mantid import plots

w=Load('CNCS_7860')
fig, ax = plt.subplots(subplot_kw={'projection':'mantid'})
ax.plot(w,LogName='ChopperStatus5')
fig.show()

fig, ax = plt.subplots(subplot_kw={'projection':'mantid'})
ax.plot(w,LogName='ChopperStatus5', StartFromLog=True)
fig.show()

fig, ax = plt.subplots(subplot_kw={'projection':'mantid'})
ax.plot(w,LogName='ChopperStatus5', FullTime=True)
fig.show()

w1=Load('HYS_13658')
fig, ax = plt.subplots(subplot_kw={'projection':'mantid'})
ax.plot(w1,LogName='s1',StartFromLog=True,FullTime=True)
fig.show()
fig, ax = plt.subplots(subplot_kw={'projection':'mantid'})
ax.plot(w1,LogName='s1')
fig.show()

fig = plt.figure()
ax1 = fig.add_subplot(211,projection='mantid')
ax2 = fig.add_subplot(212,projection='mantid')
ax1.plot(w, LogName='ChopperStatus5')
ax1.set_title('From run start')
ax2.plot(w, LogName='ChopperStatus5',FullTime=True)
ax2.title('Absolute time')
fig.tight_layout()
fig.show()
```



No issue or release notes

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
